README.md

```diff
@@ -86,6 +87,12 @@ You can jump right into editing this file [here](https://github.com/ErichDonGubl
 * Designing low latency DSP algorithms suitable for embedded use (common filters, analysis functions)
 * Library for nonlinear dynamical or chaotic systems (solvers, numeric methods etc.)
 
+#### Personal information management
+
+* Contacts via [vCard](https://en.wikipedia.org/wiki/VCard) have been implemented using [`vobject`](https://crates.io/crates/vobject), but no "high-level interface" yet exists using it or an alternative. <!-- FIXME: What does this actually mean? What APIs are missing/expected? -->
+* [iCalendar](https://en.wikipedia.org/wiki/ICalendar) parsing has been implemented via several crates (i.e., [`vobject`](https://crates.io/crates/vobject)), but a higher-level API is missing. <!-- FIXME: What does this actually mean? What APIs are missing/expected? -->
+* [`beancount`](http://furius.ca/beancount/) has no APIs written in Rust at all!
```

Comment:
I'd rather write "There is no parser/emitter/builder for beancount files for Rust" or something like that.

Reply:
Maybe I could add a small inner list? Like this:

* Parser
* Emitter
* Builder (distinction from emitter is...what? Just a nicer way to construct the data structures the emitter expects?)
* Python integration to reuse logic from the original library (?)

Reply:
That sounds nice, yes!

Sorry for the long delay, am busy with exams :smiley: